src/Reflection/ReflectedNode.php

```diff
@@ -17,10 +17,22 @@
 
 namespace Cake\ApiDocs\Reflection;
 
+use Cake\Core\Configure;
+
 abstract class ReflectedNode
 {
     public string $name;
 
+    public string $githubLink;
+
+    public array $repoMap = [
```

Comment:
This should be part of the config file.

Reply:
Sure, I can adjust that right now if you want